src/wp-admin/js/customize-controls.js

```diff
@@ -9115,7 +9158,6 @@
 
 		// Add behaviors to the static front page controls.
 		api( 'show_on_front', 'page_on_front', 'page_for_posts', function( showOnFront, pageOnFront, pageForPosts ) {
-			// eslint-disable-next-line func-style
```

Comment:
`// eslint-disable-next-line func-style`

Leave this comment line in the code to silence `Expected a function declaration` errors / warnings in ESLint.

Reply:
Thanks for the explanation! Now done.